### PR TITLE
add useGasless option

### PR DESCRIPTION
**Pull request by m-awais-ahmad**
## Pull Request Description

- add following properties to `WriteCallFunctionDto`
  - `useGasless`
  - `fee`
- add `useGasless` to `IssueTokenRequest `
- add `useGasless` to `ContractDeployRequest`

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] I have added corresponding labels to the PR

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -2100,6 +2101,7 @@ export type ParameterWithValueList = ParameterWithValue[] | ParameterWithValueLi
 interface EVMTokenCreateParamsDto {
     contractId: string;
     deployFunctionParams?: Array<ParameterWithValue>;
+    useGasless?: boolean;
```
Comment (phanigovindaraju) [inline src/types.ts:2104]: the 'useGasless' should be in IssueTokenRequest and not here I believe

File: src/types.ts [hunk 2]
```diff
@@ -2113,6 +2115,7 @@ export interface WriteCallFunctionDto {
     feeLevel?: FeeLevel;
```
Comment (phanigovindaraju) [inline src/types.ts:2115]: Looks like 'fee' is missing here. Let's add that.

File: src/types.ts (hunk 1 above)
Comment (m-awais-ahmad) [reply src/types.ts]: changed

File: src/types.ts (hunk 2 above)
Comment (m-awais-ahmad) [reply src/types.ts:2115]: added

**Review by phanigovindaraju — APPROVED** [review]

**Pull request merged**